src/librustc_resolve/diagnostics.rs

```diff
@@ -197,6 +197,66 @@ See the Types section of the reference for more information about the primitive
 types:
 
 http://doc.rust-lang.org/reference.html#types
+"##,
+
+E0364: r##"
+Private items cannot be publicly re-exported.  This error indicates
+that you attempted to write a 'pub use' against a type or value that was
```

Comment:
> attempted to `pub use` an item that was

Since these diagnostic messages eventually get parsed as markdown, inline code must be delimited by ` (U+0060 GRAVE ACCENT), not ' (U+0027 APOSTROPHE).